.changeset/fuzzy-jeans-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'explorer': minor
+---
+
+Add test IDs to the app identity section of the header.
```

Comment:
No need to add a changeset for this as it in itself does not require a release, if in adding the test you ended up fixing something you could add the bug fix to the functionality as a changeset.